spec/pins_v3_spec.rb

```diff
@@ -70,6 +70,7 @@ def add_packages
       $dut.pin[:pinz].direction.should == :io
       $dut.pin[:pinz].rtl_name.should == "pinz"
       $dut.pin[:piny].rtl_name.should == "piny[0]"
+      $dut.pin[:piny].type.should == :signal
```

Comment:
Was this nil or digital before?

Reply:
nil, `:analog` and `:digital` were the options for setting the pin type previously